docs/manual/workflow-steps/builtin.md

```diff
@@ -127,6 +127,17 @@ Finally at the moment of run our job, if the referenced job defines in Job State
 
 ### Log Data Step
 
+The log data step allows you to log all the context data values and view them in a table. 
+
+![Log Data Step](~@assets/img/log-data-set.png)
+
+Debug Only
+: If checked, it will only log data at the debug level.
```

Comment:
I think this is more specifically "Only run if the job is run in Debug mode"?  Please confirm and clarify.